modules/post/windows/gather/powershell/enum_domain_pass_policy.rb

```diff
@@ -0,0 +1,31 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'msf/core/auxiliary/report'
+
+
+class Metasploit3 < Msf::Post
+
+  include Msf::Auxiliary::Report
+
+  def initialize(info={})
+    super( update_info( info,
+        'Name'          => 'Enum Domain Password Policy',
+        'Description'   => %Q{ This module will enumerate the domain password policy },
+        'License'       => MSF_LICENSE,
+        'Author'        => [ 'Ben Turner <benpturner[at]yahoo.com>','Dave Hardy <davehardy20[at]gmail.com>'],
+        'Platform'      => [ 'win' ],
+        'SessionTypes'  => [ 'powershell' ]
+      ))
+  end
+
+  # Run Method called when command run is issued
+  def run
+    print_good("Running the post module: #{name} on: " + session.shell_command('$env:COMPUTERNAME').gsub!(/(\r\n)/, ''))
+    pscommand='net accounts /domain'
```

Comment:
This could be SessionTypes `'shell', 'powershell'` it isn't PowerShell specific and should be cross compatible if `cmd_exec` is implemented for powershell.